dll/ime/msctfime/msctfime.rc

```diff
@@ -33,6 +33,9 @@ IDI_DOWN ICON     "res/down.ico"
 #ifdef LANGUAGE_PT_PT
     #include "lang/pt-PT.rc"
 #endif
+#ifdef LANGUAGE_TR_TR
+    #include "lang/tr-TR.rc"
+#endif
 #ifdef LANGUAGE_RU_RU
     #include "lang/ru-RU.rc"
 #endif
```

Comment:
I somehow overlooked this, it's not sorted right alphabetically (T comes after R)

Reply:
Good point! I will fix it. Thx 4 mentioning.